test/datasets/test_iwslt2016.py

```diff
@@ -165,8 +167,12 @@ def test_iwslt2016(self, split, src, tgt, dev_set, test_set):
     @parameterized.expand(["train", "valid", "test"])
     def test_iwslt2016_split_argument(self, split):
         root_dir = self.get_base_temp_dir()
```

Comment:
If you are planning on removing the call to `get_base_temp_dir `, can we modify this test class to stop extending the `TempDirMixin` class?